core/modules/admin_bar/css/admin_bar.css

```diff
@@ -6,10 +6,13 @@
  *
  * @see https://www.htmldog.com/articles/suckerfish
  */
+:root {
```

Comment:
```suggestion
/* Set a CSS property declaring the height of the admin bar. */
/* This can be read by themes to offset the top of the page if needed. */
:root {
```